spec/java/util/stream/DoubleStream.lsl

```diff
@@ -45,7 +49,7 @@ import java/util/Spliterator;
 @GenerateMe
 @implements("java.util.Spliterator.OfDouble")
 @public type DoubleStreamLSLSpliterator
-    is java.util.DoubleStreamLSLSpliterator
+    is java.util.stream.DoubleStreamLSLSpliterator
```

Comment:
I'm sorry, but I can't quite parse what you mean by that. Is this a request for changes or just a side-note?

Reply:
I suggest to back this spec to package "java.util.DoubleStreamLSLSpliterator"
because original class url is in package: /**java/util**/Spliterators.java